samples/nrf9160/modem_shell/src/sms/sms.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SMS_H
+#define SMS_H
```

Comment:
Should be careful here, there already is a library in NCS under `modem/sms.h` which defines `SMS_H_`; that's very close. There are a lot of headers here, perhaps we could prefix them with `MOSH` or similar to avoid conflicts?

Reply:
I've gone through all our headers and added MOSH prefix to them.